docs/source/orderer/ordering_service.md

```diff
@@ -74,41 +74,47 @@ process with three phases that ensures all of the peers in a blockchain network
 keep their ledgers consistent with each other.
 
 In the first phase, a client application sends a transaction proposal to
-a subset of peers that will invoke a smart contract to produce a proposed
-ledger update and then endorse the results. The endorsing peers do not apply
-the proposed update to their copy of the ledger at this time. Instead, the
-endorsing peers return a proposal response to the client application. The
+the Fabric Gateway service. The target peer --- a peer in the same organization
+submitting the proposal --- then runs the transaction and forwards the proposal
+to endorsing peers in the organizations required by the endorsement policy. The
+target peer will invoke a smart contract to execute the transaction and return
+the proposed ledger update to the client, along with the endorsement results. The
```

Comment:
The prior two sentences repeat each other. First sentence says "runs the transaction and forwards the proposal". Second sentence says "invoke the smart contract" but doesn't mention forwarding it to other peers. The first sentence is the correct one.

Reply:
ok deleting second. 

To me the second adds to the first rather than repeating. We should watch out for identifying repetition based on prior knowledge that all readers may not have.